examples/lxmert/checkpoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+wget http://nlp.cs.unc.edu/models/faster_rcnn_from_caffe_attr.pkl
+mv faster_rcnn_from_caffe_attr.pkl checkpoint.pkl
+https://raw.githubusercontent.com/airsplay/py-bottom-up-attention/master/demo/data/images/input.jpg
```

Comment:
should it have a curl/wget?
```suggestion
wget https://raw.githubusercontent.com/airsplay/py-bottom-up-attention/master/demo/data/images/input.jpg
```